src/rpcrawtransaction.cpp

```diff
@@ -334,14 +334,19 @@ Value createrawtransaction(const Array& params, bool fHelp)
             "    {\n"
             "      \"address\": x.xxx   (numeric, required) The key is the smileycoin address, the value is the btc amount\n"
             "      ,...\n"
+            "    },\n"
+            "    {\n"
+            "      \"data\": \"hex:amount\"    (obj, optional) A key-value pair. The key must be \"data\", the value is hex encoded data and amount of SMLY\n"
+            "                                  to be burnt by this OP_RETURN output. Default value is 0\n"
             "    }\n"
-
+            "    ,...                   More key-value pairs of the above form. For compatibility reasons, a dictionary, which holds the key-value pairs directly, is also\n"
+            "                           accepted as second parameter.\n"
             "\nResult:\n"
             "\"transaction\"            (string) hex string of the transaction\n"
 
             "\nExamples\n"
             + HelpExampleCli("createrawtransaction", "\"[{\\\"txid\\\":\\\"myid\\\",\\\"vout\\\":0}]\" \"{\\\"address\\\":0.01}\"")
-            + HelpExampleRpc("createrawtransaction", "\"[{\\\"txid\\\":\\\"myid\\\",\\\"vout\\\":0}]\", \"{\\\"address\\\":0.01}\"")
+            + HelpExampleCli("createrawtransaction", "\"[{\\\"txid\\\":\\\"myid\\\",\\\"vout\\\":0}]\" \"[{\\\"data\\\":\\\"00010203\\\"}]\"")
```

Comment:
Why is the HelpExampleRpc() being removed? Surely we keep that as well as the new HelpExampleCli() line?

Reply:
Yes, I did not intend on removing it. Will update it quickly